src/pinentry.rs

```diff
@@ -187,6 +187,9 @@ fn parse_pinentry_pin<R>(response: R) -> anyhow::Result<String>
 where
   R: AsRef<str>,
 {
+  const OK_PREFIX: &str = "D ";
```

Comment:
Intuitively, I expected `OK_PREFIX` to be `"OK"`.  Maybe `DATA_PREFIX`?

Reply:
Good call. Will change it. Thanks for your review!